### ISSUE TITLE
Question about Wifi

### ISSUE THREAD

**Issue by vricosti** [SchmockLord/Hackintosh-Intel-i9-10900k-Gigabyte-Z490-Vision-D#218]
Hi,

I have the following hardware:
Gigabyte Z490 Vision-D (Bios F21) with Intel Wi-Fi 6 AX201
Intel(R) Core(TM) i7-10700K CPU @ 3.80GHz
Gskill DDR4 32GB
Gigabyte RX 5700 XT GAMING OC (GV-R57XTGAMING OC-8GD)
Nvme Samsung SSD 970 PRO 512GB (MacOS partition - Linux Partition)
Nvme Samsung SSD 970 PRO 1TB (Windows partition)
Crucial MX500 2To (Data partition)
OpenCore-DEBUG 0.7.8

And I have followed the guide on tonymacx86 and successfully installed Monterey (before I tried BigSur but wifi was not working) and I am successfully using the default Wifi card but on your config you seem to use another wifi card.
However some of my usb ports don't seem to work very well and at boot I have a message: cannot drop acpi ...
So my question is if I replace my EFI folder with yours will I be able to still use my wifi connection ?

**Comment by SchmockLord** [comment]
Yes you can use the Intel Wifi with my config. But then you need to disable another HSxx port and enable HS14.
